hue/hue.sh

```diff
@@ -143,6 +143,12 @@ EOF
   for interpreter in "${interpreters[@]}"; do
     sed -i "/^ *# \[\[\[${interpreter}\]\]\]/,/^ *$/s/^\( *\)# \?/\1/" /etc/hue/conf/hue.ini
   done
+  # Enable hive on 4.5, 4.8
+  local FIX_HIVE_VERSION=("4.5.0 4.8.0")
+  local HUE_VERSION=$(sed -n 's/VERSION="\(.*\)"/\1/p' /usr/lib/hue/VERSION)
+  if [[ "${FIX_HIVE_VERSION[*]}" =~ "${HUE_VERSION}" ]]; then
```

Comment:
As we associate Hue version with Dataproc versions, I think that we can just use existing `DATAPROC_VERSION` env var and apply this logic for Dataproc 1.5 and 2.0.

Reply:
Changed to DATAPROC_VERSION